cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -809,41 +809,14 @@ public void deleteFlowQueues(HttpRequest request, HttpResponder responder,
   @DELETE
   @Path("/queues")
   public void clearQueues(HttpRequest request, final HttpResponder responder) {
```

Comment:
Better rename the method to `deleteQueues` to match with what the http method is.